manifests/d/danielchalmers/SteamAccountSwitcher/2.3.1/danielchalmers.SteamAccountSwitcher.locale.en-US.yaml

```
@@ -0,0 +1,22 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: danielchalmers.SteamAccountSwitcher
+PackageVersion: 2.3.1
+PackageLocale: en-US
+Publisher: Daniel Chalmers
+PublisherUrl: https://github.com/danielchalmers
+Author: Daniel Chalmers
+PackageName: SteamAccountSwitcher

```

```suggestion
PackageName: Steam Account Switcher
```
This must match Control Panel.